adversarial_text/layers.py

```diff
@@ -76,7 +76,8 @@ def build(self, input_shape):
   def call(self, x):
     embedded = tf.nn.embedding_lookup(self.var, x)
     if self.keep_prob < 1.:
-      embedded = tf.nn.dropout(embedded, self.keep_prob)
+      shape = embedded.get_shape().as_list()
+      embedded = tf.nn.dropout(embedded, self.keep_prob, noise_shape=(shape[0], 1, shape[2]))
```

Comment:
Since embedded is [batch_size, num_timesteps, embedding_dim], this noise_shape will drop or keep entire timesteps - is that correct/intended? If so, please leave a short comment in the code explaining the noise_shape.